docs/geedocs/5.3.0/answer/7160007.html

```diff
@@ -152,6 +152,14 @@ <h5>
               Added an experimental option that fixes several memory leaks related to our usage of the Xerces library.
             </td>
           </tr>
+          <tr>
+            <td>1228</td>
+            <td>Insufficient screen real estate for 'Assemble' button</td>
```

Comment:
Let's re-word this to some thing like "`Assemble` button is not fully visible making difficult to use the `GLC assembly tool`"

Reply:
Done.

Reply:
i don't see the change. Was it pushed ?